README.rst

```diff
@@ -282,7 +282,140 @@ For more information about these options, see the `Getting Help`_ page.
 .. _Getting Help: https://openedx.org/getting-help
 
 License
-*******
+=======
+
+The LTI Consumer XBlock is available under the AGPL v3 License.
+
+Security
+========
+
+Please do not report security issues in public. Send security concerns via email to security@edx.org.
+
+Changelog
+=========
+
+Please See the [releases tab](https://github.com/edx/xblock-lti-consumer/releases) for the complete changelog.
+
+4.2.0 - 2022-05-11
+------------------
+* Refactor anonymous user to real user rebinding function to use `rebind_user` service.
+* Refactor accessing hostname from runtime attribute to using `hostname` service.
```

Comment:
@xitij2000, it seems that we're maintaining the changelog in the `Changelog.rst` file.